src/main/java/com/sparta/vicky/comment/entity/Comment.java

```diff
@@ -0,0 +1,83 @@
+package com.sparta.vicky.comment.entity;
+
+import com.sparta.vicky.baseEntity.Timestamped;
+import com.sparta.vicky.comment.dto.CommentRequest;
+import com.sparta.vicky.user.entity.User;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Getter
+@NoArgsConstructor
+public class Comment extends Timestamped {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "comment_id")
+    private Long id;
+
+    @Column(nullable = false)
+    private String content;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "user_id")
+    private User user;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "board_id")
+    private Board board;
+
+    @@Column(nullable = false)
+    private Long likes;
+
+    /**
+     * 생성자
+     */
+    public static Comment createComment(CommentRequest request, Board board, User user) {
+        Comment comment = new Comment(request.getContent());
+        comment.setBoard(board);
+        comment.setUser(user);
+
+        return comment;
+    }
+
+    private Comment(String content) {
+        this.content = content;
+    }
```

Comment:
`likes` 필드를 0으로 초기화해주세요.
```suggestion
    private Comment(String content) {
        this.content = content;
        this.likes = 0;
    }
```